Automattic/wp-calypso
# Add connect card in domain-only settings page

## Changes proposed in this Pull Request

This PR add a new card in domain settings page for parked domain (aka domain-only domains).

It's part of the new domain-only flow (pcYYhz-ye-p2)

![domain-only-settings-new](https://user-images.githubusercontent.com/2797601/154689133-95a25d92-b9e2-41e7-9448-c2f9a764ee10.png)

## Testing instructions

_Prerequisites: you need a domain-only subscription to test this PR_

- Build this branch locally or open the live Calypso link
- Navigate to to `/manage/domains` and select a parked domain
- Verify that the domain settings page show the Status card as default
- Verify that the two buttons lead to _create site_ and _add domain to site_ flows.
